docs/LSPApi.md

```diff
@@ -511,6 +511,54 @@ public class MyLSPSemanticTokensFeature extends LSPSemanticTokensFeature {
 }
 ```
 
+### Semantic Tokens File View Providers

```

Comment:
This now includes a simple API by which custom language server integrations against non-TextMate (and non-plain text) files can leverage the semantic token information. I've documented how and why to use that API here.